nucypher/config/keyring.py

```diff
@@ -576,9 +557,7 @@ def generate(cls,
             signing_private_key, signing_public_key = _generate_signing_keys()
 
             if checksum_address is FEDERATED_ADDRESS:
-                uncompressed_bytes = signing_public_key.to_bytes(is_compressed=False)
-                without_prefix = uncompressed_bytes[1:]
-                verifying_key_as_eth_key = EthKeyAPI.PublicKey(without_prefix)
+                verifying_key_as_eth_key = EthKeyAPI.PublicKey.from_compressed_bytes(bytes(signing_public_key))
```

Comment:
As you already know, this module has been completely reworked in #2701.  Leaving this comment as a note-to-self to revisit this area.